src/Microsoft.Sbom.Targets/SbomInputValidator.cs

```diff
@@ -95,8 +98,8 @@ public EventLevel ValidateAndAssignVerbosity()
                 return EventLevel.Critical;
             default:
                 Log.LogWarning($"Unrecognized verbosity level specified. Setting verbosity level at Verbose");
```

Comment:
nit: Replace "Setting verbosity level at Verbose" to "Setting verbosity level at Information"

Reply:
Addressed at e968302eefdfd075248b6a9aa6c4eb0e7694b4b3